docs/configuration.rst

```diff
@@ -237,7 +237,19 @@ See :ref:`the corresponding tutorial <theming>`.
 Installing extra xblocks and requirements
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Would you like to include custom xblocks, or extra requirements to your Open edX platform? Additional requirements can be added to the ``env/build/openedx/requirements/private.txt`` file. For instance, to include the `polling xblock from Opencraft <https://github.com/open-craft/xblock-poll/>`_::
+Would you like to include custom xblocks, or extra requirements to your Open edX platform? Additional requirements can be added to the ``OPENEDX_EXTRA_PIP_REQUIREMENTS`` parameter in the :ref:`config file <configuration>` or to the ``env/build/openedx/requirements/private.txt`` file. For instance, to include the `polling xblock from Opencraft <https://github.com/open-craft/xblock-poll/>`_:
+
+- add the following to the ``config.yml``::
+
+    OPENEDX_EXTRA_PIP_REQUIREMENTS:
+    - "git+https://github.com/open-craft/xblock-poll.git"
+
+.. warning::
+   Specifying extra requirements through ``config.yml`` overwrites the default
+   extra requirements. You might need to add them to the list, if your
+   configuration depends on them.
```

Comment:
Here, you should link to the default values that you will document in https://docs.tutor.overhang.io/configuration.html#open-edx-customisation